src/openapi/streaming/serializer/schemas/proto-base.js

```diff
@@ -0,0 +1,70 @@
+// Stores core schemas used by all schemas received from openapi.
+
+export const protoBclBase = `package bcl;
```

Comment:
Probably not needed if we decide to go with pure google types without C# ones like Decimal.

Reply:
Yeap, removed.